test/frontend/OfficerDB.test.js

```diff
@@ -0,0 +1,23 @@
+/* global describe it */
+import 'jsdom-global/register'
+import React from 'react'
+import Enzyme, { mount } from 'enzyme'
+import { expect } from 'chai'
+
+import OfficerManager from '../../src/Pages/OfficerDB/App.js'
+import Adapter from 'enzyme-adapter-react-16'
+import { Table } from 'reactstrap'
+
+Enzyme.configure({ adapter: new Adapter() })
+
+describe('Officer Manager: <Table />', () => {
```

Comment:
please keep the same [naming convention](https://github.com/SCE-Development/Core-v4/blob/7eb49330115efe0619c97377f7b69382cca2d41d/test/frontend/Slideshow.test.js#L13) for React tests.
```suggestion
describe('<OfficerManager />', () => {
```